src/cli.rs

```diff
@@ -0,0 +1,76 @@
+use crate::{CliArgs, Client, GenesisDetails, StateId};
+use ethereum_consensus::{phase0::mainnet::Fork, primitives::Root};
+use url::Url;
+
+pub async fn cli(args: CliArgs) {
```

Comment:
nitpick on name, something more actionable like
```suggestion
pub async fn run_cli(args: CliArgs) {
```